vtex-apps/product-details
# Add ProductHighlights

#### What is the purpose of this pull request?
Improve the way information is arranged.

#### What problem is this solving?
Working with the ecowater site we perceive that has some infos that they renders which that can improve our product details. 


#### How should this be manually tested?
You should access the [storefront](https://highlightsprod--storecomponents.myvtex.com/admin/cms/storefront) and search for the `shirt` product.  Then open the schema for `ProductDetails` and change the content and analyze the behavoir. 

This PR has dependecy with: https://github.com/vtex-apps/store-graphql/pull/225, https://github.com/vtex-apps/store-theme/pull/125 and https://github.com/vtex-apps/store-components/pull/364

#### Screenshots or example usage
___Model from ecowater__ :_ 
![modelws](https://user-images.githubusercontent.com/17649410/54364104-00b71200-464b-11e9-9280-d74ce37c8d76.PNG)


#### Types of changes
